### DISCUSSION TITLE
No signing certificate "iOS Development" found

### DISCUSSION THREAD

**Discussion by hgeorgilas** [codemagic-ci-cd/codemagic-docs#2259] [Q&A]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Ionic/Cordova

### Steps to reproduce

Signing identities are uploaded correctly.

The production ad_hoc build works as expected.

Starting an app store production iOS build fails always with:
> "No signing certificate "iOS Development" found"

### Expected results

The build should finish successfully and the IPA should be exposed in the "artifacts" section or uploaded directly to the appstore account.

### Actual results

Consistently getting this error
> "No signing certificate "iOS Development" found"

### Build id (optional)

657820b17bc4a45da386d5a8

**Comment by himesh-cm** [comment]
Hi @hgeorgilas, The discussion is being continued on intercom, i will keep the thread updated here.

**Comment by icarusdust** [comment]
I am closing the thread for now, @himesh-cm will update the thread.

**Comment by himesh-cm** [comment]
Please note you are already using Code-signing identity as below in your yaml within `ios_signing:`
​
The following steps are unnecessary because they are already handled automatically.
​
```
​keychain initialize
​keychain add-certificates
```
